caffe2/python/operator_test/heatmap_max_keypoint_op_test.py

```diff
@@ -30,8 +30,8 @@ def heatmap_approx_keypoint_ref(maps, rois):
 
 def c10_op_ref(maps, rois):
     keypoints = torch.ops._caffe2.HeatmapMaxKeypoint(
-        torch.Tensor(maps),
-        torch.Tensor(rois),
+        torch.tensor(maps),
+        torch.tensor(rois),
```

Comment:
The main question I have for reviewing these `Tensor -> tensor` changes is: the dtype of the output can change from `float32` (the default for `Tensor` with e.g. a list of integers) to `int64`. It's not straightforward to see whether that matters or not. Did you check whether `HeatmapMaxKeypoint` behaves the same, or that all callers of `c10_op_ref` don't feed in integers here?

Reply:
Yes, I verified the callers of `c10_op_ref` (which, apparently is only one):

https://github.com/pytorch/pytorch/blob/f2b4b0e9eb670ad79f4460c0d45174f879b523f9/caffe2/python/operator_test/heatmap_max_keypoint_op_test.py#L136-L141

Which calls this function with an expanded version of the `input`:

https://github.com/pytorch/pytorch/blob/f2b4b0e9eb670ad79f4460c0d45174f879b523f9/caffe2/python/hypothesis_test_util.py#L647